src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,176 @@
+package duke;
+
+import duke.task.Task;
+
+/**
+ * This class is responsible for anything related to the user interface, from printing output to reading input from the
+ * user
+ */
+public class Ui {
+    private static final String HORIZONTAL_RULE = "____________________________________________________________";
+    private static final String HELLO_MESSAGE = "Hello, and welcome to my humble abode. \n"
+            + "I'm supposed to help you but I'll be the judge of that. Anyway what do you want.";
+    private static final String BYE_MESSAGE = "Thank you for that utter waste of time.\n"
+            + "Can't wait to see you again...";
+
+    /**
+     * Prints out a string which gives a greeting to the user
+     */
+    public void displayGreeting() {
+        System.out.println(getGreetingResponseAsString());
+    }
+
+    /**
+     * Get's bot's greeting as a String
+     * @return String representing bot's opening greeting
```

Comment:
need line separation between description and @return/@param (for other javadoc also)